bgpd/bgp_pbr.c

```diff
@@ -2030,6 +2030,7 @@ static void bgp_pbr_policyroute_add_to_zebra_unit(struct bgp *bgp,
 	if (bpf->type == BGP_PBR_IPRULE) {
 		memset(&temp4, 0, sizeof(temp4));
 		temp4.vrf_id = bpf->vrf_id;
+		temp4.priority = 1;
```

Comment:
I'm not a fan of putting priority of 1 here, this means that no-one can add a custom rule between that and the local lookup.  I would suggest a value of say 20, or even better would to allow the end user to customize this as that they may have other things besides bgp's flowspec adding rules

Reply:
set to 20 for now